src/components/NavLink.tsx

```diff
@@ -81,8 +81,8 @@ export function NavLink(props: NavLinkProps) {
   );
 }
 
-export function getNavLinkStyles(variant: NavLinkVariant, contrast: boolean = false) {
-  return navLinkVariantStyles(contrast)[variant];
+export function getNavLinkStyles(variant: NavLinkVariant, contrast?: boolean) {
+  return navLinkVariantStyles(!!contrast)[variant];
```

Comment:
**thought:** Can we make `contrast` a required param? We can default the value to `false` when we destructure the NavLinkProps on line 26.

Reply:
Sure, right away!